Rakefile

```
@@ -73,8 +73,8 @@ end
 desc "Clean build, node_modules and lock files"
 task :clean do
   all_apps.each do |app|
-    run_command "cd frameworks/#{app} && rm -rf build"
-    run_command "cd frameworks/#{app} && rm -rf dist"
+    config = app_config(app)
+    run_command "cd frameworks/#{app} && rm -rf #{config[:build_dir]}"
```

```suggestion
    run_command "cd frameworks/#{app} && rm -rf #{config.fetch(:build_dir)}"
```

Just in case someone forgets to specify the build_dir, let's fail when the key isn't specified.

---

Changed it throughout the application to always use `.fetch` from the config instead of silently doing the wrong thing.